package.json

```diff
@@ -77,7 +77,7 @@
     "hooks": {
       "commit-msg": "commitlint -E HUSKY_GIT_PARAMS",
       "pre-push": "yarn build-tests-changed && yarn split-e2e-tests",
-      "pre-commit": "yarn verify-commit"
+      "pre-commit": "./scripts/package-json-check.sh && yarn verify-commit"
```

Comment:
We are changing this from being reported in the CI to being fixed before the changes are committed.

Reply:
If someone runs with --no-verify, the changes can get to CI but hopefully not get into dev branch